.github/workflows/double_ws_export.yml

```diff
@@ -22,10 +22,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
If 3.8 not supported anymore, change the README documents as well for the same.

Reply:
Keras 3.0 drops python 3.8 support, but other frameworks (i.e. PyTorch) still support it. If we decide to remove support for Python 3.8, it should be it's own PR.

Reply:
raised PR https://github.com/securefederatedai/openfl/pull/1189 for version change